Week1/homework/js-exercises/ex5.js

```diff
@@ -0,0 +1,25 @@
+//Exercise 5: Round a number and log it
+//Declare variable z and assign the number 7.25 to it
+let z = 7.25;
+//log the value of z
+console.log(z);
+//Declare variable a and assign z.round value
+let a = Math.round(z);
+//log the value of a
+console.log(a);
+//compare the two values and store the highest of the two in a new variable
+if (a > z) {
+  let x = a;
+} else {
+  let x = z;
+  console.log(x);
+}
+//console.log
+function compare(a, z) {
```

Comment:
Very nice that you already use a function!
However, it does not do exactly what the instructions asked for. The instructions ask to assign the highest value to a new variable and then log that variable. 
Hint: look up `Math.max`.

Reply:
changes done locally.